src/YearsView.js

```diff
@@ -31,16 +31,25 @@ var DateTimePickerYears = React.createClass({
         var currentYear, disabled,
             // Month and date are irrelevant here because
             // we're only really interested in the year
-            irrelevantMonth = 1,
+            irrelevantMonth = 0,
```

Comment:
I am in the process of merging this commit using my own commit, but I don't understand why this change was made. Care to explain? :)